tests/unit/amo/components/TestCategory.js

```diff
@@ -436,6 +438,86 @@ describe(__filename, () => {
     });
   });
 
+  it('renders the themes shelves with the ADDON_TYPE_THEMES_FILTER filter if static theme is enabled', () => {
+    _categoriesFetch();
+    _categoriesLoad();
+    _getLanding();
+    _loadLanding();
+
+    const fakeConfig = getFakeConfig({ enableStaticThemes: true });
+
+    const root = render({ _config: fakeConfig }, { autoDispatchCategories: false });
+
+    const landingShelves = root.find(LandingAddonsCard);
+
+    expect(landingShelves.at(0)).toHaveProp('footerLink', {
+      pathname: `/search/`,
+      query: {
+        addonType: ADDON_TYPE_THEMES_FILTER,
```

Comment:
Ideally we would have something like the above but for some reason these tests are failing when I pass in this fakeConfig. It seems enableStaticTheme still comes up as false. Any idea what i am missing?

Reply:
I think the problem is that the `Category` component does not receive a `config` object, hence it cannot be passed to the `getAddonTypeFilter()` function.

Reply:
yep, I already tried it that way too and it does the same. I mean if you do it outside of here, it returns what you expect (`ADDON_TYPE_THEMES_FILTER`) value but in the query it still returns the value `ADDON_TYPE_THEME`:

![Alt text](https://monosnap.com/image/HoWAN6mzqLV6d1919zrjQlx2cv8I5M.png)

Reply:
I am not sure to follow. If `Category` accepts a `_config` prop, defaulting to `config`, and `geAddonTypFilter()` is called with `this.props._config` as a second argument, I don't understand why it would not work (if `_config` is passed to the `render` function in the test case).

Reply:
you're right! I was passing in a 2nd arg here (at 1 point) but not as a prop so it didn't work this way. thank you 👍